lib/client.js

```diff
@@ -797,6 +797,14 @@ class Client {
   }
 
   /**
+   * @name formUrlEncode
+   * @api private
+   */
+  formUrlEncode(value) {
+    return encodeURIComponent(value).replace(/%20/g, '+');
```

Comment:
- can `querystring.escape` be used instead of encodeURIComponent?
- what is the point of the replace?

I'd really appreciate tests for this and some references to validate the behavior.